### PR TITLE
Add a Django web app letting users join communities from a website

### LINKED ISSUE #23: Join communities from a website
Create a Django app - 

- List the communities
- Enable the user they can join any communities using the `join community!` button
- Deploy on heroku account.
- Submit a PR.
- Take my crendentials.
- Test the deployment on my heroku account.

The design should only have community list on one side and `join community!` button on different side.

### PR DESCRIPTION
**Pull request by singalhimanshu**
Fix #23

### CODE REVIEW

File: templates/index.html [hunk 1]
```diff
@@ -0,0 +1,135 @@
+<!DOCTYPE html>
+<html>
+    <html lang="en">
+    <head>
+		<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css" integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm" crossorigin="anonymous">
+		<script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/js/bootstrap.min.js" integrity="sha384-JZR6Spejh4U02d8jOt6vLEHfe/JQGiRRSQQxSfFWpi1MquVdAyjUar5+76PVCmYl" crossorigin="anonymous"></script>
+        <title>Communities for Women</title>
+        <meta charset="UTF-8">
+        <meta name="description", content="List of communities for empowering women">
+        <meta name="author", content="Mridu Bhatnagar">
```
Comment (tapaswenipathak) [inline templates/index.html]: Write your name.

File: templates/index.html [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+        <meta http-equiv="Content-Type" content="text/html">
+    </head> 
+    <body>
+	    <h1>Women Communities</h1>
+		<br>
+        <table class="table table-striped">
+            <thead>
+				<tr>
+				<th scope="col">List of Women Communities in STEM</th>
+				<th scope="col">Join Community</th>
```
Comment (tapaswenipathak) [inline templates/index.html]: All link should open in separate tab.

File: templates/index.html [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+				</tr>
+			</thead>
+			<tbody>
+				<tr>
+					<td>community.anitab.org</td>
+					<td>
+						<a class="btn btn-primary" href="https://community.anitab.org" role="button">Join Community</a>
```
Comment (tapaswenipathak) [inline templates/index.html]: link should open in separate tab.

File: Procfile [hunk 4]
```diff
@@ -0,0 +1 @@
+web: gunicorn list.wsgi
```
Comment (tapaswenipathak) [inline Procfile:1]: %s/`list.wsgi`/`womenstemcommunities.wsgi`.

File: .gitignore [hunk 5]
```diff
@@ -0,0 +1,6 @@
+*.pyc
+db.sqlite3
+__pycache__
+local_settings.py
+.vscode
```
Comment (tapaswenipathak) [inline .gitignore]: No code editor specific files in `.gitignore`.

**Review by tapaswenipathak — COMMENTED** [review]
I wrote few comments, please squash the commits, looks great!

File: Procfile (hunk 4 above)
Comment (singalhimanshu) [reply Procfile:1]: Didn't understand this one.

Comment (tapaswenipathak) [reply Procfile]: In vim you `:%s` when replacing text. I will do it.

Comment (tapaswenipathak) [reply Procfile:1]: In vim you `:%s` when replacing text. I will do.

**Pull request merged**